graph_nets/modules.py

```diff
@@ -118,14 +118,16 @@ def __init__(self,
           use_globals=False,
           received_edges_reducer=reducer)
 
-  def _build(self, graph):
+  def _build(self, graph, edge_kw={}, node_kw={}):
```

Comment:
Maybe use "edge_model_kwargs" and "node_model_kwargs", to keep the same language as in the constructor.

(Here and in other modules)